examples/ethfs/src/FileStore.ts

```diff
@@ -15,15 +18,17 @@ ponder.on("FileStore:FileCreated", async ({ event, context }) => {
 
   const metadata = parseJson(fromHex(rawMetadata as Hex, "string"));
 
-  await context.entities.File.create({
+  await context.models.File.create({
     id: filename,
     data: {
       name: filename,
       size: Number(size),
-      contents: await context.contracts.FileStoreFrontend.read.readFile([
-        event.transaction.to as `0x{string}`,
-        filename,
-      ]),
+      contents: await context.client.readContract({
+        abi: FileStoreFrontendAbi,
+        functionName: "readFile",
+        address: "0xBc66C61BCF49Cc3fe4E321aeCEa307F61EC57C0b",
+        args: [event.transaction.to!, filename],
+      }),
```

Comment:
I guess this works!! Seems like a fair enough pattern.

Reply:
Gets tough when you want to use multichain but I wanted to get this moving.